Client/Frontend/Autofill/CreditCard/CreditCardBottomSheet/CreditCardBottomSheetViewController.swift

```diff
@@ -266,6 +266,8 @@ class CreditCardBottomSheetViewController: UIViewController, UITableViewDelegate
         hostingCell.contentView.backgroundColor = .clear
         hostingCell.selectionStyle = .none
         hostingCell.isAccessibilityElement = true
+        let accessibilityCCNumberLast4 = creditCard.ccNumberLast4.map { String($0) }.joined(separator: " ")
+        hostingCell.accessibilityLabel = "\(creditCard.ccType), \(creditCard.ccName), \(creditCard.ccType) ••••\(accessibilityCCNumberLast4), Expires \(creditCard.ccExpMonth)/\(creditCard.ccExpYear)"
```

Comment:
This would need to be localized as well. And on another note: We already have a localized string for something that is very similar and we should probably use it here as well: `.CreditCard.Settings.ListItemA11y`.

Reply:
I've realized it after working on a different ticket for which I made a PR today, will modify accordingly